testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -193,6 +195,67 @@ public void testCreateSchema()
         assertUpdate("DROP SCHEMA " + schemaName);
     }
 
+    @Test
+    public void testRenameTable()
+    {
+        if (!hasBehavior(SUPPORTS_RENAME_TABLE)) {
+            // must reference an existing table in ALTER TABLE since name is validated before calling renameTable
```

Comment:
nit: IMO comment unnecessary (but doesn't harm)

Reply:
I would remove the comment.

Reply:
Removed